website/cue/reference/remap/functions/del.cue

```diff
@@ -15,6 +15,13 @@ remap: functions: del: {
 			required:    true
 			type: ["path"]
 		},
+		{
+			name:        "compact"
+			description: "If compact is true, after deletion, if an empty object or array is left behind, it should be removed as well, cascading up to the root. This only applies to the path being deleted, and any parent paths."
```

Comment:
nit: can we split this up into multiple lines? (unfortunately, we don't have a line-length lint for Cue docs as we do for Markdown docs).